libs/eo/shared/atomic-design/feature-molecules/src/lib/eo-popup-message/eo-popup-message.component.ts

```diff
@@ -108,30 +106,34 @@ import { MatIconModule } from '@angular/material/icon';
       <img alt="Danger icon" src="/assets/icons/danger.svg" />
       <div class="content">
         <h4 class="watt-space-stack-s">Oops! Something went wrong...</h4>
-        <p class="watt-space-stack-m">
+        <p>
           We apologize for the inconvenience, please reload the page or try
-          again. <br />
+          again.
+        </p>
+        <p class="watt-space-stack-m">
           If you still experience issues, you are welcome to contact customer
           service, and we will help you proceed.
         </p>
-        <p class="inline">
+        <div style="display: flex; justify-content: space-between">
+          <p class="inline">
+            <a
+              href="mailto:datahub@energinet.dk?subject=Error on energioprindelse.dk&body=**Write your message here, but please keep the lines below intact for our internal investigations**%0D%0A%0D%0AError code:{{
+                errorMessage?.status
+              }}%0D%0A{{ errorMessage?.message }}"
+              >Contact customer service</a
+            >
```

Comment:
Is this way of handling errors agreed with any of the POs (Søren or Andreas)?

Reply:
Sorry, I missed that this is only presentation changes, not adding more to the error popup